exercises/react/example/Reactor.ceylon

```diff
@@ -0,0 +1,25 @@
+class Reactor<Element>() {
+  shared interface Cell {
+    shared formal Element currentValue;
+  }
+
+  shared class InputCell(Element initialValue) satisfies Cell {
+    variable Element currentValue_ = initialValue;
+    shared actual Element currentValue => currentValue_;
+    assign currentValue {
+      currentValue_ = currentValue;
+    }
+  }
+
+  shared class ComputeCell satisfies Cell {
+    variable Element currentValue_;
+    Element() newValue;
+
+    shared new single(Cell c, Element(Element) f) {
```

Comment:
Because it was not true, I was forced to give up making this generic. Now our reactor only acts on Integers.